odoo/addons/base/models/res_lang.py

```diff
@@ -32,7 +33,9 @@ class Lang(models.Model):
     active = fields.Boolean()
     direction = fields.Selection([('ltr', 'Left-to-Right'), ('rtl', 'Right-to-Left')], required=True, default='ltr')
     date_format = fields.Char(string='Date Format', required=True, default=DEFAULT_DATE_FORMAT)
+    date_format_preview = fields.Char(string='Date format preview', compute="_compute_date_format_preview", readonly=True, store=False)
```

Comment:
this is already the default for a computed field

(same below)
```suggestion
    date_format_preview = fields.Char(string='Date format preview', compute="_compute_date_format_preview")